src/index.ts

```diff
@@ -37,7 +37,18 @@ export const createAppAuth: StrategyInterface = function createAppAuth(
     },
     options.installationId
       ? { installationId: Number(options.installationId) }
-      : {}
+      : {},
+    {
+      log: Object.assign(
+        {
+          debug: () => {},
+          info: () => {},
+          warn: console.warn.bind(console),
+          error: () => {},
```

Comment:
as we only use `log.warn`, I'd simply remove the other methods 
```suggestion
          warn: console.warn.bind(console),
```